catalog/spatial/geocoding/spatial-geocoding-offline-index/pom.xml

```diff
@@ -116,6 +116,7 @@
                         </Embed-Dependency>
                         <Import-Package>
                             !com.fasterxml.jackson*,
+                            !org.noggit,
```

Comment:
This fixes an existing issue (that was caused by a previous dependency upgrade if i recall correctly)

Reply:
I think this got inlined into Solrj.  Looks like I forgot to export it from `solr-dependencies` since most places were excluding it like this.

https://github.com/apache/lucene-solr/blob/62f969403ae445910b580079d40f6abf714705aa/solr/solrj/src/java/org/noggit/package-info.java